<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>421</width>
        <height>681</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>371</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>可用设备</string>
       </property>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>271</width>
          <height>109</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/3.1.png);</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>77</width>
          <height>83</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_openBLE">
           <property name="text">
            <string>打开蓝牙</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_closeBLE">
           <property name="text">
            <string>断开连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_upDateBLE">
           <property name="text">
            <string>刷新设备</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>381</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);</string>
       </property>
       <property name="title">
        <string>发送区</string>
       </property>
       <widget class="QTextEdit" name="btn_text2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_setJudge">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>31</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>更</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_clear2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>80</y>
          <width>31</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>清屏</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>接收温度</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>最低温：</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Read_tohex">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>最高温：</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="W_Max">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_sendtest">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_lengthClear">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>清屏</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="setJudge">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>90</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>基于QT+单片机的温度监控系统</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/3.1.png</normaloff>:/3.1.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(138, 155, 173, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_receive">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clearReceive">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清屏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>接受区</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_file">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>文件</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
